bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/package.scala

```diff
@@ -8,4 +11,30 @@ package object core {
   type Dec = Declination
   val  Dec = Declination
 
-}
\ No newline at end of file
+  type Ephemeris = Long ==>> Coordinates
```

Comment:
Would it make sense to us an actual `Date` (please the Java 1.8 date) instead of `Long`?